whois-api/src/main/java/net/ripe/db/whois/api/apiKey/BearerTokenExtractor.java

```diff
@@ -24,84 +22,95 @@
 
 import javax.annotation.Nullable;
 import java.net.URI;
-import java.net.URL;
-import java.util.Arrays;
-import java.util.HashSet;
 
 @Component
 public class BearerTokenExtractor   {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BearerTokenExtractor.class);
 
     private final boolean enabled;
-    private final String whoisKeycloakId;
-    private final String jwksSetUrl;
+    private final URI tokenIntrospectEndpoint;
+    private final ClientSecretBasic keycloakClient;
 
     @Autowired
     public BearerTokenExtractor(@Value("${apikey.authenticate.enabled:false}") final boolean enabled,
-                                @Value("${api.public.key.url:}")  final String jwksSetUrl,
+                                @Value("${openId.metadata.url:}")  final String openIdMetadataUrl,
```

Comment:
We need to remember to add this property to the environment properties file (and in the deployment notes for the next Whois release).

Reply:
I updated the release notes